dynamic-config.md

```diff
@@ -132,7 +132,6 @@ The following TiKV configuration items can be modified online:
 | `raftstore.raft-log-gc-count-limit` | The hard limit on the allowable number of residual Raft logs |
 | `raftstore.raft-log-gc-size-limit` | The hard limit on the allowable size of residual Raft logs |
 | `raftstore.raft-entry-cache-life-time` | The maximum remaining time allowed for the log cache in memory |
-| `raftstore.raft-reject-transfer-leader-duration` | Determines the smallest duration that a Leader is transferred to a newly added node |
```

Comment:
Is this deprecated since v5.0.0?

Reply:
Yes.

Reply:
So why remove a config (deprecated in v5.0.0) in v5.3, not in the v5.0, v5.1 and v5.2? Also, https://github.com/tikv/tikv/pull/11249 was merged only half a month ago.

Reply:
Because it was forgot before v5.3. And it was found half a month ago.